features/web-share.yml

```
@@ -1,7 +1,8 @@
-draft_date: 2024-10-28
-name: Web Share API
-description: TODO
+name: Web Share
+description: The `navigator.share()` method invokes the native sharing mechanism of the underlying operating system and passes text, links, files, and other content to share targets.
```

https://github.com/web-platform-dx/web-features/blob/main/docs/guidelines.md#device

```suggestion
description: The `navigator.share()` method invokes the device's native sharing mechanism and passes text, links, files, and other content to share targets.
```